en-US/P.xml

```diff
@@ -109,12 +112,11 @@
 			<term>plug-in</term>
 			 <listitem>
 				<para>
-					<emphasis>n.</emphasis> Write hyphenated. Do not use "plugin".
+					<emphasis>n.</emphasis> Write hyphenated.
+          Do not use "plugin".
```

Comment:
Although some product terminology uses "plugin", and the CCS Supplementary Style Guide now defines the unhyphenated form as the standard.